src/main/java/com/iridium/iridiumskyblock/commands/BanCommand.java

```diff
@@ -0,0 +1,108 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumcore.utils.StringUtils;
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.PermissionType;
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.IslandBan;
+import com.iridium.iridiumskyblock.database.IslandTrusted;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumskyblock.gui.BansGUI;
+import com.iridium.iridiumskyblock.utils.PlayerUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.Player;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * Command which bans a visitor from your island
+ */
+public class BanCommand extends Command {
+
+    /**
+     * The default constructor.
+     */
+    public BanCommand() {
+        super(Arrays.asList("ban", "bans"), "Ban a player visiting from your island", "%prefix% &7/is ban [<name>]", "", true, Duration.ZERO);
+    }
+
+    /**
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     * Bans a player visiting from an island
+     *
+     * @param sender The CommandSender which executes this command
+     * @param args   The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public boolean execute(CommandSender sender, String[] args) {
+        if (args.length != 2 && args.length != 1) {
```

Comment:
um? cant we just do if args.length > 2 ?

Reply:
for 0 ?